ts/client/src/accounts/group.ts

```diff
@@ -90,7 +90,9 @@ export class Group {
     public vaultAmountsMap: Map<string, BN>,
   ) {}
 
-  public async reloadAll(client: MangoClient, ids?: Id): Promise<void> {
+  public async reloadAll(client: MangoClient): Promise<void> {
```

Comment:
removing the ids option, not all pathways were using ids json before, now once configured on the client during building the client, the gpa vs api is used implicitly